src/loaders/configure/nonce.js

```diff
@@ -0,0 +1,22 @@
+/* global __webpack_require__ */
+
+__webpack_require__.nc = (() => {
+  try {
+    if (document.currentScript && document.currentScript.nonce) {
+      return document.currentScript.nonce
+    }
+
+    // Fallback for browsers that do not support document.currentScript (IE11)
+    const scriptTags = document.querySelectorAll('script')
+    for (let i = 0; i < scriptTags.length; i++) {
+      const scriptTag = scriptTags[i]
+      if (scriptTag.nonce) {
+        return scriptTag.nonce
+      }
+    }
+  } catch (ex) {
+    // Swallow error and proceed like nonce is not defined
+  }
```

Comment:
This assumes that the first script is the loader, are we good with that assumption?  Would we be better off just not trying to support IE11?

Reply:
For IE11, we are assuming that any script containing a `nonce` will work because that script would otherwise fail if that `nonce` was not in the CSP. I feel like this is a safe assumption.